### ISSUE TITLE
use monkberry in Node Express JS app?

### ISSUE THREAD

**Issue by zenithtekla** [monkberry/monkberry#29]
How to set monkberry to render view engine in Node Express app?

**Comment by elfet** [comment]
Monkberry is now suitable only for web rendering. 

But i have plans for adding server side rendering.

**Comment by vamshi9** [comment]
Hope this helps you![https://github.com/monkberry/monkberry/issues/2](url)

**Comment by zenithtekla** [comment]
Thank you for your inputs. I gave a quick look at your reference and saw there ain't app.set('view engine', 'monkberry') or the like.

To be more specific about my question
Assume, as in [here](https://github.com/zenithtekla/nodeMySQL/blob/master/modules/core/server/configs/core.server.configs.js#L50), I loaded the routes with 
    `require(routes)(app);`

**Do I need** to use app.engine() like the commented part? **How to declare** it correctly for Monkberry instead of Swig?
If I don't need app.engine(), **do I declare** it with `app.set('view engine', 'monkberry');` ? or something, by the way, **what is the extension** for monkberry template files (html, mbr, mkb, or what)? for `hoganjs`, it's `hjs` and you name the files `*.hjs` , for `pug` and `jade` (jade is deprecated), it's `pug` and you name it `.pug`

```
    /* swig & handlebars require engine to be declared
     app.engine('html', swig.renderFile);
     */
    
    app.set('views', view_path);
    app.set('view engine', 'hjs');
```

Thank you very much.
